flink-table-store-core/src/test/java/org/apache/flink/table/store/file/utils/BlockingIterator.java

```diff
@@ -65,6 +65,15 @@ public BlockingIterator(Iterator<IN> iterator, Function<IN, OUT> converter) {
         }
     }
 
+    public List<OUT> collectAndCloseQuietly(int limit) {
```

Comment:
Why do you want to be quiet? Just throw the exception?

Reply:
> Why do you want to be quiet? Just throw the exception?

because the `collect()` is direct returns an empty list when the expected num is zero. What I want to test is to let time pass by until timeout, to verify that it cannot collect even one record.

Reply:
It'll be more dedicated and straightforward  if moving this method to `ReadWriteTableITCase`